### ISSUE TITLE
Skip existing records in db on bulkinser?

### ISSUE THREAD

**Issue by raskolnikoov** [zzzprojects/Dapper-Plus#4]
Is it possible to use BulkInsert and if the record already exists in the db it skips to insert that record. Lets say my table has a customerid and companyid. If A record with existing values wants to be inserted we skip the insertion for this record. Is it possible?

**Comment by JonathanMagnan** [comment]
Hello @raskolnikoov ,

The version v1.0.15 has been released:
https://www.nuget.org/packages/Z.Dapper.Plus/

We added a new property in the mapping to allow this logic. At this moment, it can only be done in the mapping.

```csharp
DapperPlusManager.Entity<Order>()
                 .InsertIfNotExists();

connection.BulkInsert(orders);
```

See: https://github.com/zzzprojects/Dapper-Plus/wiki/dapper-plus-mapper-action

Let us know if this is what you was looking for and if you successfully implement it.

Best Regards,

Jonathan
